docs/get-started.md

```diff
@@ -26,7 +26,9 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
 
 ### Set up server
 
-1. Start the server and the local storage service. In the root directory of Ark, run:
+These instructions set up an Ark server that lets you test only the most basic `ark create backup` and `ark create restore` commands. For the additional configuration you need to get logs or run `ark describe` commands, see the following section.
```

Comment:
The first sentence reads a little strangely to me.  Maybe something like "These instructions start  the Ark server and a Minio instance that is accessible from within the cluster only.  To be able to access logs and run `ark describe` commands, you need to be able to access Minio from outside the cluster.  See the following section..."